sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -196,3 +200,25 @@ def test_v_measure_and_mutual_information(seed=36):
         assert_almost_equal(v_measure_score(labels_a, labels_b),
                             2.0 * mutual_info_score(labels_a, labels_b) /
                             (entropy(labels_a) + entropy(labels_b)), 0)
+
+
+def test_max_n_classes():
```

Comment:
You need to create a local random number generator:
<pre>
rng = np.random.RandomState(seed=0)
</pre>
and use it after:
<pre>
labels_true = rng.random(53)
</pre>